.github/workflows/tests.yml

```diff
@@ -0,0 +1,11 @@
+name: CI
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install modules
+        run: npm install
```

Comment:
not sure whether you're using yarn or npm, as there are both yarn.lock and package-lock.json files in repo